Assets/GothicVR/Scripts/Creator/MusicCreator.cs

```diff
@@ -42,6 +42,13 @@ public enum Tags : byte
 
         private static GameObject backgroundMusic;
 
+        private float updateInterval = 5f; // Update music every 5 seconds
+        private float timer = 0f;
+
+        // This multiplier is used to increase the buffer size and reduce the number timess PrepareData is called
+        // also affects the delay of the music, it doesnt sound so harsh when switching
+        private int bufferSizeMultiplier = 16;
```

Comment:
Why 16? Can it be calculated from sth. like updateInterval * x?

Reply:
from manual testing this gives the music a big enough buffer so it sound good when switching tracks